multirom/multirom.cpp

```diff
@@ -46,6 +48,9 @@ MROMInstaller *MultiROM::m_installer = NULL;
 MultiROM::baseFolders MultiROM::m_base_folders;
 int MultiROM::m_base_folder_cnt = 0;
 bool MultiROM::m_has_firmware = false;
+std::string    MultiROM::tmp_str = "";
+std::string   MultiROM::MR_UPDATE_SCRIPT_PATH ="META-INF/com/google/android/";
```

Comment:
Don't do this, it's useless to allocate static std::strings. The error you were getting is because c++x11 expects a space in between, like this: 

`"/tmp/" MR_UPDATE_SCRIPT_PATH `